juego.py

```diff
@@ -66,87 +70,101 @@ def __init__(self):
         pass
 
 
+    global sx, sy
+    def sx(coord_x):
+        return coord_x * scale_x
+
+    def sy(coord_y):
+        return coord_y * scale_y
+
     def main(self):
-        sonido_menu = load_sound("./data/menu.ogg")
+        sonido_menu = load_sound("menu.ogg")
         jugar = self.fuente_130.render("JUGAR",True,(0,0,255))
         level = self.fuente_130.render("NIVEL",True,(0,0,255))
         quit = self.fuente_130.render("SALIR",True,(0,0,255))
         fondo = cargar_imagen('data/1.jpg')
         chosen_level = "facil"
 
-        while 1:
+        while self.running:
             self.screen.fill((0,0,0))
             self.screen.blit(fondo, (0, 0))
-            self.screen.blit(jugar,(450,100))
-            self.screen.blit(level,(450,200))
-            self.screen.blit(quit,(450,300))
+            self.screen.blit(jugar,(sx(450), sy(100)))
+            self.screen.blit(level,(sx(450), sy(200)))
+            self.screen.blit(quit,(sx(450), sy(300)))
```

Comment:
mixing comma whitespace styles makes this harder to read; i don't think it is time to adopt space after comma.

Reply:
Sure, I'll change them.

Reply:
Done in d1e8746